packages/core/src/themes/buildTheme.ts

```diff
@@ -82,7 +82,7 @@ export default function buildTheme(
       },
       disabled: {
         opacity: disabledOpacity,
-        cursor: 'normal',
+        cursor: 'default',
```

Comment:
This is a big change throughout.

Reply:
yeah I'll pull this out into a different PR.